src/libp2p/connectionsManager.ts

```diff
@@ -14,25 +14,27 @@ import { ZBAY_DIR_PATH } from '../constants'
 import fs from 'fs'
 import path from 'path'
 import { IChannelInfo } from '../storage/storage'
-import fetch from 'node-fetch';
+import fetch from 'node-fetch'
 import debug from 'debug'
-import CustomLibp2p from './customLibp2p'
+import CustomLibp2p, { Libp2pType } from './customLibp2p'
 const log = Object.assign(debug('waggle:conn'), {
   error: debug('waggle:conn:err')
 })
 
-interface IOptions {
-  env?: {
-    appDataPath: string
-  },
-  bootstrapMultiaddrs?: Array<string>
+class IOptions {
+  env: {
+    appDataPath?: string
+  } = {}
+  bootstrapMultiaddrs: Array<string> = []
```

Comment:
let's try to be consistent and use either Array<string> or string[] across the whole project, but not both